src/argilla/client/sdk/v1/datasets/api.py

```diff
@@ -249,11 +225,30 @@ def add_records(
     response = client.post(url=url, json={"items": records})
 
     if response.status_code == 204:
-        return Response(
-            status_code=response.status_code,
-            content=response.content,
-            headers=response.headers,
-        )
+        return Response.from_httpx_response(response)
+    return handle_response_error(response)
+
+
+def delete_records(
+    client: httpx.Client, id: UUID, record_ids: List[UUID]
+) -> Response[Union[ErrorMessage, HTTPValidationError]]:
+    """Sends a DELETE request to `/api/v1/{id}/records` endpoint to remove a list of `FeedbackDataset` records.
+
+    Args:
+        client: the authenticated Argilla client to be used to send the request to the API.
+        id: the id of the dataset to remove the records from.
+        record_ids: the IDs of the records to be removed from the dataset.
+
+    Returns:
+        A `Response` object with the response itself, and/or the errors codes if applicable.
+    """
+    url = f"/api/v1/datasets/{id}/records"
+
+    uuids_str = ",".join([str(record_id) for record_id in record_ids])
+    response = client.delete(url=url, params={"ids": uuids_str})
```

Comment:
This may introduce unexpected in proxied environments. If the URL is too large, the client can receive a 414 status error. The default nginx configuration (the standard proxy setup) is 8K, making the `ds.delete_records` useless